types/jsreport-core/index.d.ts

```diff
@@ -15,20 +18,36 @@ declare namespace JsReport {
     type Recipe = "html";
 
     interface Template {
-        content: string;
+        content?: string;
         engine: Engine | string;
-        helpers: Helpers;
+        helpers?: Helpers;
         recipe: Recipe | string;
+        pathToEngine?: string;
     }
 
-    interface Options {
+    interface RequestOptions {
         preview?: boolean;
+        timeout?: number;
+        reportName?: string;
     }
 
     interface Request {
-        template: Partial<Template>;
-        options?: Options;
-        data: any;
+        value?: boolean;
+        writable?: boolean;
+        configurable?: boolean;
+        enumerable?: boolean;
+        template?: Partial<Template>;
+        options?: RequestOptions;
+        context?: Context;
+        data?: any;
```

Comment:
`data` is not required? Could you show the doc about this here?

Reply:
https://github.com/jsreport/jsreport-core/blob/b3f907cc3b8a4fe18dc91f48430b2df6e691264c/lib/render/request.js#L38-L41

Reply:
Well done.